2001-3000/LIT2222RetuaH.xml

```diff
@@ -33,7 +34,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <langUsage><language ident="en">English</language></langUsage>
+         <langUsage>
+           <language ident="en">English</language>
+           <language ident="gez">Gǝʿǝz</language>
+         </langUsage>
+         <textClass>
+            <keywords>
+               <term key="ChristianLiterature"/>
+               <term key="Homily"/>
+            </keywords>
+         </textClass>
+<abstract>
+<p>Rǝtuʿa Hāymānot is a collection of homilies focusing on the events of the Passion cycle. Most probably it is an indigenous
+  Ethiopian production. Its as author must have lived ca. 1375 and there is some ground to identify him
+  with <persName ref="PRS4709Giyorgis">Giyorgis of Saglā</persName>. It should not be confused with a collection of homilies
+preserved in <ref type="mss" corresp="EMML1763">EMML 1763</ref>, <ref type="mss" corresp="EMML8509">EMML 8509</ref>
```

Comment:
If there would be an ID for this other collection, the users would be happy to find it, but I assume it doesn't exist yet.

Reply:
No it doesn't. I hope at one point to distinguish between various persons called RH and collections.